mvp-spec/signatures.md

```diff
@@ -0,0 +1,62 @@
+# Signatures
+
+A quick introduction on how data (id, preferences) and messages (operator API, RTB) are signed and how these signatures can be verified.
+
+PAF Data format is designed to let the users audit how their preferences got
+to their current state. It relies on the signatures of data and communication, to enforce security.
+
+All "signers" have an Elliptic Curve (EC) **private** and **public** key using the `secp256r1` curve.
```

Comment:
Important: there is not one private and public key. There is one public key and one private key.

Wording question: is it common to say an "EC key"? We often speak about the algorithm instead (ECDSA)

Reply:
Of course, the sentence reads "one private AND ONE public" but I will rephrase to avoid confusion, thanks.

I'm not sure about the wording, I will try a better wording after some more reading on the topic 😅

Reply:
After some research, I believe the wording is correct in the sense that the keys define what elliptic curve will be used.

Reply:
reword pushed, please close the conversation if you agree.

Reply:
Can you share the source of your reading, please ? When I google "Elliptic Curve key", it doesn't exit.

Reply:
For example:
https://cryptobook.nakov.com/asymmetric-key-ciphers/elliptic-curve-cryptography-ecc
https://avinetworks.com/glossary/elliptic-curve-cryptography/

My point is that (to my understanding) keys are **generated for** ECC, they are dedicated to this type of encryption so they can't be used in a different context. This is why I believe they can be called ECC keys because they need to be created for this algorithm.

Reply:
I may be nitpicking but your sources comfort my intuition:  "ECC key" makes sense because the Elliptic Curve is related to Cryptography. However, EC Key doesn't appear in the source that you shared.